contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -945,6 +1013,12 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
       bytes[] memory destExecDataPerToken
     )
   {
+    // Prevents a stack too deep error on messageData
+    {
+      bytes memory data = messageData;
+      (convertedExtraArgs, isOutOfOrderExecution) = _processChainFamilySelector(destChainSelector, data, extraArgs);
```

Comment:
Only the length is ever used, it's a lot cheaper to pass in a single length property